packages/docs/src/pages/components.js

```diff
@@ -49,12 +100,19 @@ const ComponentsPage = ({ data }) => {
   const componntsGroupList = map(componentsGrouped, (components, componentGroupName) => (
     <ComponentGroup key={slugify(componentGroupName)} groupName={componentGroupName} components={components} />
   ));
+  // Generate side menu using component-group-component-name
+  const componentSideMenuItems = map(componentsGrouped, (components, componentGroupName) => (
+    <ComponentSideMenu key={slugify(componentGroupName)} groupName={componentGroupName} components={components} />
+  ));
 
   return (
     <main className="docs-masthead">
       <h1 className="sr-only">Components Page</h1>
       <div className="container">
```

Comment:
Should we change this to `container-fluid`?

Reply:
line 111? Not sure stretching the whole page looks good?

Reply:
idk, trying to avoid the crunch (~990px, which is fairly wide) as it has room, but labels start wrapping. Maybe we just need some custom styles

![image](https://user-images.githubusercontent.com/664714/38051758-37cb3108-329d-11e8-9612-59483243a13f.png)